axon/neuromod.go

```diff
@@ -85,10 +85,10 @@ const (
 type NeuroModParams struct {
 	DAMod       DAModTypes  `desc:"effects of dopamine modulation on excitatory and inhibitory conductances"`
 	DAModGain   float32     `viewif:"DAMod!=NoDAMod" desc:"multiplicative factor on overall DA modulation specified by DAMod -- resulting overall gain factor is: 1 + DAModGain * DA, where DA is appropriate DA-driven factor"`
-	DALrateMod  slbool.Bool `desc:"modulate learning rate as a function of abs(DA) absolute value of dopamine"`
-	DALratePct  float32     `min:"0" max:"1" viewif:"DALrateMod" desc:"proportion of maximum learning rate that DA can modulate -- e.g., if 0.2, then DA = 0 = 80% of std learning rate, 1 = 100%"`
-	AChLrateMod slbool.Bool `desc:"modulate learning rate as a function of ACh acetylcholine level"`
-	AChLratePct float32     `min:"0" max:"1" viewif:"AChLrateMod" desc:"proportion of maximum learning rate that ACh can modulate -- e.g., if 0.2, then ACh = 0 = 80% of std learning rate, 1 = 100%"`
+	DALRateMod  slbool.Bool `desc:"modulate learning rate as a function of abs(DA) absolute value of dopamine"`
```

Comment:
from the description, it seems like we don't need these bools; Can't we just set DALRatePct to 0 to disable modulation?

Reply:
yeah, I often like to have a bool so you can preserve the param value for reference, but in this case, I think it does make sense to just go with the 0 value to turn off.

Reply:
done